### PR TITLE
Adding members to Security Response Team

### PR DESCRIPTION
**Pull request by varun-lodaya**
### Description
This change creates a new Security Response Team and adds core security maintainers to it

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by varun-lodaya** [comment]
@dblock @peternied can you'll review this?

**Comment by varun-lodaya** [comment]
GH issue  - https://github.com/opensearch-project/.github/issues/219

File: SECURITY.md [hunk 1]
```diff
@@ -8,22 +8,25 @@ If you discover a potential security issue in this project we ask that you notif
 
 *Giving credit where credit is due, this policy is heavily influenced by the [Xen Project’s security response process](https://xenproject.org/developers/security-policy/), that was put to the test during the [embargo period for XSA-108 back in 2014](https://xenproject.org/2014/10/22/xen-project-security-policy-improvements-get-involved/) and improved its clarity around managing the pre-disclosure list and the deployment of fixes during embargo. We are standing on the shoulders of these battle-tested giants.*
 
-## The Security Team
+## The Security Response Team (SRT)
```
Comment (dblock) [inline SECURITY.md:11]:
```suggestion
## Security Response Team (SRT)
```

File: SECURITY.md [hunk 2]
```diff
@@ -8,22 +8,25 @@ If you discover a potential security issue in this project we ask that you notif
...
 
-The OpenSearch Security Team is a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. Maintainers can be added to the Security Team by submitting a PR updating this document and adding their name to the list below (the process for becoming a maintaner can be found [here](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer)).
+The OpenSearch Security Response Team (SRT) comprises of a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. New SRT members can only be nominated by current SRT members.
```
Comment (dblock) [inline SECURITY.md:13]:
```suggestion
The OpenSearch Security Response Team (SRT) is comprised of a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. New SRT members are nominated by current SRT members.
```

**Review by dblock — APPROVED** [review]
LGTM, I made some nits if you want to iterate more.

**Review by peternied — APPROVED** [review]
Thanks @varun-lodaya for the update - thanks also to the new community members that are actively engaged in the security response process for the project.

I like @dblock suggestions and would recommend you take them.

**Review by varun-lodaya — COMMENTED** [review]
Addressing nitpic comments from dbock@

**Review by dblock — APPROVED** [review]

**Review by peternied — APPROVED** [review]

**Comment by varun-lodaya** [comment]
Can somebody help merge this?

**Review by saratvemulapalli — APPROVED** [review]

**Review by dblock — APPROVED** [review]
Will hit merge when bake time is done.

**Pull request merged**